cluster-up/cluster/ocp-4.3/provider.sh

```diff
@@ -44,7 +44,11 @@ function up() {
     user=$(echo $auth |awk -F: '{print $1}')
     password=$(echo $auth |awk -F: '{print $2}')
 
-    ${_cli} run okd ${params} --container-registry-user $user --container-registry-password $password
+    if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
+      ${_cli} run okd ${params} --container-registry-user $user --container-registry-password $password --container-registry ${container_registry}
```

Comment:
I don't understand the code here, but it is better to set
```
if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
   params="$params --container-reg $container-registry"
fi
```
over writing a long line twice. Code duplication is evil.

Reply:
tried that at first
it cant work that way because you need --container-reg "" explicit
unless there is a way to do that

Reply:
```
else
    params="$params --container-reg ''"
```

Reply:
tried that before and also now
it gives this instead of just kubevirtci/ocp-4.3-provision:latest
2019/12/29 09:14:56 failed to download ''/kubevirtci/ocp-4.3-provision:latest: invalid reference format

Reply:
params=" --container-registry= $params"
this might work
tested it dry and it seems fine, but better to test it fully on next provision
meanwhile if its ok i will current code ?

Reply:
Additionally the indentation looks astray from the default here.

Reply:
fixed both issues